BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/Skonto/DocumentPages/DocumentPagesViewController.swift

```diff
@@ -15,6 +15,23 @@ final class DocumentPagesViewController: UIViewController {
         return view
     }()
 
+    private lazy var navigationBar: UINavigationBar = {
+        let navBar = UINavigationBar()
+        navBar.translatesAutoresizingMaskIntoConstraints = false
+        navBar.setBackgroundImage(UIImage(), for: .default)
+        navBar.shadowImage = UIImage()
+        navBar.isTranslucent = true
+        navBar.backgroundColor = .GiniBank.dark1.withAlphaComponent(0.5)
+        navBar.titleTextAttributes = [.font: configuration.textStyleFonts[.bodyBold] as Any,
+                                             .foregroundColor: UIColor.white]
```

Comment:
[request] it should be .light1 for text based on Figma

Reply:
Fixed!